wrappers/windres-wrapper.sh

```diff
@@ -1,3 +1,183 @@
+  -v, --verbose              Enable verbose output.
+  -V, --version              Display version.
+  -h, --help                 Display this message and exit.
+Input Formats:
+  rc                         Text Windows Resource
+  res                        Binary Windows Resource
+Output Formats:
+  res                        Binary Windows Resource
+  coff                       COFF object
+Targets:
+  armv7-w64-mingw32
+  aarch64-w64-mingw32
+  i686-w64-mingw32
+  x86_86-w64-mingw
+EOF
+    exit 1
+}
+
+error() {
+    echo "${PROG}: error: $1" >&2
+    exit 1
+}
+
+quote() {
+    echo "$1" | sed -e "s|'|'\\\\''|g; 1s/^/'/; \$s/\$/'/"
+}
+
+INCLUDE=
+VERBOSE=${VERBOSE:-false}
+INPUT=-
+OUTPUT=/dev/stdout
+INPUT_FORMAT=rc
+OUTPUT_FORMAT=res
+TARGET=X64
+CODEPAGE=
+
+while [ $# != 0 ]; do
+    case "$1" in
+        "-i="*|"--input="*)
+            INPUT="${1#*=}";;
+        "-i"|"--input")
+            INPUT="${2}"; shift;;
+        "-o="*|"--output="*)
+            OUTPUT="${1#*=}";;
+        "-o"|"--output")
+            OUTPUT="${2}"; shift;;
+        "-J="*|"--input-format="*)
+            INPUT_FORMAT="${1#*=}";;
+        "-J"|"--input-format")
+            INPUT_FORMAT="${2}"; shift;;
+        "-O="*|"--output-format="*)
+            OUTPUT_FORMAT="${1#*=}";;
+        "-O"|"--output-format")
+            OUTPUT_FORMAT="${2}"; shift;;
+        "-F="*|"--target="*)
+            TARGET="${1#*=}";;
+        "-O"|"--output-format")
+            TARGET="${2}"; shift;;
+        "-I"|"--include-dir")
+            INCLUDE="${INCLUDE} ${1#*=}"
+            ;;
+        "-I="*|"--include-dir="*)
+            INCLUDE="${INCLUDE} ${2}"; shift
+            ;;
+        "-c"|"--codepage")
+            CODEPAGE="${1#*=}"
+            ;;
+        "-c="*|"--codepage="*)
+            CODEPAGE="${2}"; shift
+            ;;
+        "--preprocessor")
+            error "ENOSYS"
+            ;;
+        "--preprocessor-arg")
+            error "ENOSYS"
+            ;;
+        "-v"|"--verbose")
+            VERBOSE=true
+            ;;
+        "-V"|"--version")
+            print_version;;
+        "--help"|"-h")
+            print_help;;
+        "-"*)
+            error "unrecognized option: \`$1'"
+            ;;
+        *)
+            error "rip: \`$1'"
+    esac;
+    shift
+done
+
+if $VERBOSE; then
+    set -x
+fi
+
+
+TMPDIR="$(mktemp -d /tmp/windres.XXXXXXXXX)" || error "couldn't create temp dir"
+
+case "${INPUT_FORMAT}" in
+    "rc")
+        $CC -E -P -xc -DRC_INVOKED=1 "${INPUT}" -o "${TMPDIR}/post.rc" || error "preprocessor failed"
+        sed '/^#/d;/^$/d' "${TMPDIR}/post.rc" > "${TMPDIR}/in.rc"
+        case "${OUTPUT_FORMAT}" in
+            "res")
+                llvm-rc "${TMPDIR}/in.rc" /FO "${TMPDIR}/out.res"
+                cat "${TMPDIR}/out.res" > "${OUTPUT}"
+                ;;
+            "coff")
+                llvm-rc "${TMPDIR}/in.rc" /FO "${TMPDIR}/in.res"
+                llvm-cvtres "${TMPDIR}/in.res" /OUT:"${TMPDIR}/out.o"
```

Comment:
This call is missing specifying the machine to build for with a `/machine:` option. (Or is that perhaps even required to be uppercase?)